### PR TITLE
use strings slices from k8s utils

### PR DESCRIPTION
**Pull request by krishnaindani**
## Description

Changes proposed in this pull request:

- Use slices functions for Kubernetes utils package

## Related issue(s)

Fixes https://github.com/capactio/capact/issues/411

### CODE REVIEW

**Comment by lukaszo** [comment]
Hi, @krishnaindani thank you for your contribution. This part looks ok.

There should be similar check in the `build.go` file but it seems it's missing. Can you add it also there?https://github.com/capactio/capact/blob/main/internal/cli/capact/build.go#L125 You should iterate over `Images.All()` and check if `names` contain `name`

**Review by lukaszo — COMMENTED** [review]
Hi, @krishnaindani thank you for your contribution. This part looks ok.

There should be similar check in the `build.go` file but it seems it's missing. Can you add it also there?https://github.com/capactio/capact/blob/main/internal/cli/capact/build.go#L125 You should iterate over `Images.All()` and check if `names` contain `name`

**Comment by krishnaindani** [comment]
> Hi, @krishnaindani thank you for your contribution. This part looks ok.
> 
> There should be similar check in the `build.go` file but it seems it's missing. Can you add it also there?https://github.com/capactio/capact/blob/main/internal/cli/capact/build.go#L125 You should iterate over `Images.All()` and check if `names` contain `name`

sure, I will take a look at it.

**Comment by krishnaindani** [comment]
@lukaszo I have updated that function, let me know how that looks.

File: internal/cli/capact/build.go [hunk 1]
```diff
@@ -122,14 +123,14 @@ func DeleteImages(images []string) error {
 func BuildImages(w io.Writer, repository, version string, names []string) ([]string, error) {
 	var created []string
 
-	for _, name := range names {
-		image, ok := Images[name]
+	for _, image := range Images.All() {
+		ok := slices.Contains(names, image)
 		if !ok {
-			return nil, fmt.Errorf("cannot find image %s", name)
+			return nil, fmt.Errorf("cannot find image %s", image)
```
Comment (lukaszo) [inline internal/cli/capact/build.go:129]: if `image` is not in the `names` just continue. Don't return error

**Review by lukaszo — APPROVED** [review]
Looks good. Thanks :+1:  :)

**Comment by krishnaindani** [comment]
Thank you

**Pull request merged**